File: src/segundaParteEjerciciosTraduccion/ejercicio5/ejercicio5.java
package segundaParteEjerciciosTraduccion.ejercicio5;

import java.util.Scanner;

public class ejercicio5 {
    public static void main(String[] arg){
        try{
            Scanner scanner = new Scanner(System.in);
            System.out.println("Dime un numero");
            int numero = Integer.parseInt(scanner.nextLine());
            scanner.close();
            for(int i = 0;i<=10;i++){
                System.out.println(numero + " X " + i + " = " + numero*i);
            }
        }catch (NumberFormatException e){
            System.out.println("ni pa eso sirves");
        }
    }
}

File: src/segundaParteEjerciciosTraduccion/ejercicio5/ejercicio5.kt
package segundaParteEjerciciosTraduccion.ejercicio5

// 5. Tabla de multiplicar
// Pedir un número al usuario e imprimir su tabla de multiplicar
// del 1 al 10.
fun main(){
    println("dime un numero:")
    val numero = readln().toIntOrNull()
    if(numero != null){
        for (i in 1..10){
            println("$numero X $i = ${numero * i}")
        }
    }else{
        println("no sirve ni pa eso")
    }
}